atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraService.java

```diff
@@ -65,4 +70,32 @@ public static synchronized void stop() {
         daemon.deactivate();
         daemon = null;
     }
+
+    private static void waitForDaemonToStart() throws InterruptedException {
```

Comment:
This was sadness.  No matter what I did with awaitility I couldn't get it to ever be able to connect to cassandra.